### PR TITLE
feat: TSK-15 - Initialize project

### PR DESCRIPTION
**Pull request by federicorispo**
- Added pre-commit, licenses and Consul configuration files
- Added files for the deb/rmp package
- Added bootstrap for initializing the database
- Added Jenkinsfiles
- Add catalog.info file for Backstage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+See [COPYING](COPYING) file for the project license details
+
+### Copyright notice
+
+All non-software material (such as, for example, names, images, logos, sounds) is owned by Zextras
+s.r.l. and is licensed under [CC-BY-NC-SA](https://creativecommons.org/licenses/by-nc-sa/4.0/).
+
+Where not specified, all source files owned by Zextras s.r.l. are licensed under AGPL-3.0-only
+
+[contributors-badge]: https://img.shields.io/github/contributors/zextras/carbonio-user-management-sdk "Contributors"
```
Comment (RakuJa) [inline README.md]: Remove usermanagement reference

File: README.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+[contributors]: https://github.com/zextras/carbonio-user-management-sdk/graphs/contributors "Contributors"
+
+[activity-badge]: https://img.shields.io/github/commit-activity/m/zextras/carbonio-user-management-sdk "Activity"
```
Comment (RakuJa) [inline README.md]: Remove usermanagement reference

File: README.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+[activity]: https://github.com/zextras/carbonio-user-management-sdk/pulse "Activity"
```
Comment (RakuJa) [inline README.md]: Remove usermanagement reference

File: README.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+</div>
+
+## How to install 🏁
+
+### Preparing the environment
+
+- Install PostgreSQL: `apt install postgresql`
+- Create a `postgres` superuser:
+  ```bash
+  # sudo -u postgres psql
```
Comment (RakuJa) [inline README.md]: Why are the instruction commented out?

**Review by RakuJa — CHANGES_REQUESTED** [review]
Update readme

File: README.md (hunk 4 above)
Comment (federicorispo) [reply README.md]: It is not commented out: the bash indicates that the command must run as root

**Review by RakuJa — APPROVED** [review]

**Pull request merged**